src/hooks/base.js

```diff
@@ -5,7 +5,7 @@ export default class extends EventEmitter {
         document.addEventListener(eventName, function (e) {
             for (let target = e.target; target && target !== this; target = target.parentNode) {
                 if (target.matches(elementSelector)) {
-                    handler.call(target, e);
+                    handler.call(target, e, target);
```

Comment:
so we pass target 2 times now ?

Reply:
should the first argument be `selector` ?

Reply:
first param is for binding `this` context